gdbflash.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
-cd $1/build
+if [ $# -ne 2 ]; then
+    echo "Please enter two arguments (Project name and Port number)"
+    exit 1
+fi
 
-arm-none-eabi-gdb $1.elf -ex "target extended-remote /dev/ttyACM$2" -ex 'monitor tpwr enable' -ex 'monitor swdp_scan' -ex 'attach 1' -ex 'load'
+
+project=$1
+port=$2
+
+if [ -d $project ]; then
+	if [ ${project: -1} == '/' ]; then 
```

Comment:
Is this a check to see if the input ends in a `/`?

I don't think that's necessary, I could be wrong but I think `cd EpsilonLights//build` does the same thing as `cd EpsilonLights/build`